### PR TITLE
Update the recipe

### PR DESCRIPTION
**Pull request by GregoryAshton**
1) Add extra dependencies used by gwpy for data fetching
2) Add missing command line programs

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by GregoryAshton** [comment]
<code>@<space/>conda-forge-admin, please rerender</code>

**Comment by duncanmmacleod** [comment]
@conda-forge-admin, please rerender.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,6 +17,9 @@ build:
     - bilby_pipe_generation = bilby_pipe.data_generation:main
     - bilby_pipe_analysis = bilby_pipe.data_analysis:main
     - bilby_pipe_create_injection_file = bilby_pipe.create_injections:main
+    - bilby_pipe_pp_test = bilby_pipe.pp_test:main
+    - bilby_pipe_plot = bilby_pipe.plot:main
+    - bilby_pipe_gracedb = bilby_pipe.gracedb:main
```
Comment (duncanmmacleod) [inline recipe/meta.yaml]: @GregoryAshton, does this script have a dependency on `ligo-gracedb`?

Comment (GregoryAshton) [reply recipe/meta.yaml]: Yes it does, we already have this in the requirements right?

Comment (duncanmmacleod) [reply recipe/meta.yaml]: Right you are, sorry, I'm not awake properly yet.

**Comment by GregoryAshton** [comment]
@duncanmmacleod, thanks I had noticed that but forgot to push the change to this branch.

**Pull request merged**